frontend/src/components/orgFacilities/ManageOrganization.vue

```diff
@@ -31,6 +31,14 @@
               {{ organizationModel.legalName }}
             </v-col>
           </v-row>
+          <v-row v-if="organizationModel.incNumber" dense>
```

Comment:
Technically the requirement is that it should show for a Non-Profit or Registered company. Given this is a required field in both portal (Application) and CRM this is effectively the same thing but if the required requirement changes or we run into bad data it might be safer to check the type. This is the logic from Application.

`    hasIncorporationNumber() {
      return [ORGANIZATION_TYPES.NON_PROFIT_SOCIETY, ORGANIZATION_TYPES.REGISTERED_COMPANY].includes(
        this.organizationModel.organizationType,
      );
    },`

Reply:
Technically anything other than a non-profit or a registered company should not have an incorporation number, which was my train of thought here. I'll go get the logic

Reply:
Thanks. Having the explicit logic just makes it a little clearer if someone opens the code in the future.